emoji_client.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+import slack
+
+
+client = slack.WebClient(token=os.environ['SLACK_API_TOKEN'])
```

Comment:
so if I 
```py
import emoji_client
```
then this remains private and `emoji_client.add_emoji`, etc. are exposed?  Or is this exposed as a global in the importing file?

Reply:
this would be exposed. I've changed the structure up a bit so that an instance of the client is exposed